src/faker.ts

```diff
@@ -530,6 +530,74 @@ export class Faker {
       'This method has been removed. Please use the constructor instead.'
     );
   }
+
+  /**
+   * Creates a new instance of faker with the same state as this instance.
```

Comment:
```suggestion
   * Creates a new instance of Faker with the same state as this instance.
```

I would name it with capital F, because Faker is the class and faker would be a variable/instance of Faker